src/app/modules/practitioner-registration/pages/practitioner-attachment/practitioner-attachment.component.ts

```diff
@@ -101,23 +91,14 @@ export class PractitionerAttachmentComponent extends BcpBaseForm implements OnIn
       return null;
     }
 
-    if (this.dataService.attachmentEffectiveDate
-      && this.dataService.attachmentCancelDate
-      && (isAfter(this.dataService.attachmentEffectiveDate, this.dataService.attachmentCancelDate)
-        || isSameDay(this.dataService.attachmentEffectiveDate, this.dataService.attachmentCancelDate)
-      )) {
-      return this.dataService.attachmentEffectiveDate;
+    if (this.dataService.attachmentEffectiveDate) {
+      return isAfter(this.dataService.attachmentEffectiveDate, this.dataService.facEffectiveDate)
+        ? this.dataService.facEffectiveDate
+        : this.dataService.attachmentEffectiveDate;
```

Comment:
const _effectDate = manualReview ? bcpProgramStartDate : facEffectiveDate;
if (this.dataService.attachmentEffectiveDate)
return isAfter(this.dataService.attachmentEffectiveDate, _effectDate ) ? _effectDate  : attachmentEffectiveDate;